packages/communication-react/src/mergedHooks.ts

```diff
@@ -143,3 +145,37 @@ export const usePropsFor = <Component extends (props: any) => JSX.Element>(
     throw 'Could not find props for this component, ensure the component is wrapped by appropriate providers.';
   }
 };
+
+/**
+ * Helper function to pull the necessary properties for a calling component. There is a general usePropsFor
+ * function however since it can pull in for both calling and chat you will not be able to leverage any treeshaking
+ * functionality.
+ *
+ * @public
+ */
+export const useCallingPropsFor = <Component extends (props: any) => JSX.Element>(
+  component: Component
+): ComponentProps<Component> => {
+  const props = useCallingPropsForInternal(component);
+  if (props === undefined) {
+    throw 'Could not find props for this component, ensure the component is wrapped by appropriate providers.';
```

Comment:
throw new error instead of just throw "string", otherwise it doesn't contain the stacktrace
```suggestion
    throw new Error('Could not find props for this component, ensure the component is wrapped by appropriate providers.');
```

same with other fn